src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -68,17 +68,6 @@
 
   public StashRepository(@Nonnull Job<?, ?> job, @Nonnull StashBuildTrigger trigger) {
     this(job, trigger, null);
-  }
-
-  // For unit tests only
```

Comment:
This is weird, I though we already have fmt-maven-plugin active
Why do we still get those formatting changes?

Reply:
That was just a patch rendering. I'm changing this PR to make the difference easier to see.